celo-org/celo-monorepo
# [Attestation Bot] Fix choice of number, and US number creation

### Description

Attestation Bot fixes:
* Pick only numbers that can receive incoming SMS (have a dummy URL specified for Twilio to POST to)
* Create numbers in the US correctly  
* Add explicit Twilio dependency for celotool

Baklava deployment fixes:
* deploy latest to baklava
* no longer require Twilio messaging service ID for Attestation Bot (API calls that used it had no effect)
* update helm charts

### Tested

Locally and by deploying to baklava

### Backwards compatibility

Yes
